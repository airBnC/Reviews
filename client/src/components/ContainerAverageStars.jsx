import React from 'react';

import TotalReviews from './TotalReviews';
import SearchReviews from './SearchReviews';
import ReviewStarsByDescriptor from './ReviewStarsByDescriptor';

const ContainerAverageStars = props => (
  <div>
    <h3> Average Stars Container </h3>
    <TotalReviews />
    <SearchReviews />
    <ReviewStarsByDescriptor />
  </div>
);

export default ContainerAverageStars;
