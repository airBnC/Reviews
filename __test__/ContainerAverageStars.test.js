import React from 'react';
import { configure, shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ContainerAverageStars from '../client/src/components/ContainerAverageStars';

configure({ adapter: new Adapter() });

describe('ContainerAverageStars', () => {
  it('should render correctly', () => {
    const output = shallow(<ContainerAverageStars />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
