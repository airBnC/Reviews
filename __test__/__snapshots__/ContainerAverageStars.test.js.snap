// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerAverageStars should render correctly 1`] = `
<styled.div>
  <styled.span>
    <SubcontainerAverageStarsHeader
      currentSearch=""
      handleSearchChange={[Function]}
      handleSearchSubmit={[Function]}
      numReviews={1}
      overallAvg={1}
    />
  </styled.span>
  <ReviewStarsByDescriptor
    accuracyAvg={1}
    checkinAvg={1}
    cleanlinessAvg={1}
    communicationAvg={1}
    locationAvg={1}
    valueAvg={1}
  />
</styled.div>
`;
